app/lib/evss/claims_service.rb

```diff
@@ -0,0 +1,33 @@
+module EVSS
```

Comment:
do we want to autoload and eager_load each individual directory that we add to lib?